### PR TITLE
fix: stop reinstalling all casks on every run

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

File: environments/all/Brewfile [hunk 1]
```diff
@@ -83,10 +87,12 @@ cask 'quicklook-csv'   # preview csvs
 cask 'qlvideo'         # preview more video types
 
 # Install App Store packages
-mas 'Keynote', id: 409_183_694
+# for som reason, Keynote, Numbers, and Pages insist on being reinstalled on
+# everr run, so, for the time being, they've been disabled.
```
Comment (rimarsh) [inline environments/all/Brewfile]:
```suggestion
# for some reason, Keynote, Numbers, and Pages insist on being reinstalled on
# every run, so, for the time being, they've been disabled.
```

Comment (rimarsh) [reply environments/all/Brewfile]: Hm, darn, the suggestion didn't take for multiple lines.

Comment (ianwremmel) [reply environments/all/Brewfile]: were you just trying to correct "everr" to "every"? or was there more wrong there?

Comment (rimarsh) [reply environments/all/Brewfile]: Also “for som reason”

Comment (ianwremmel) [reply environments/all/Brewfile]: Fixed

**Pull request merged**